Feature/footer

MajaTrof opened the pull request:
change container

MajaTrof committed:
html

MajaTrof committed:
container

change to 1200px

FrontKid commented on src/sass/1-helpers/_common.scss:
```diff
@@ -14,17 +14,17 @@ body {
 
 .container {
-  max-width: 1168px;
+  max-width: 1200px;
```
Верни контейнер

MajaTrof replied:
вернула

FrontKid commented on src/sass/1-helpers/_common.scss:
```diff
@@ -14,17 +14,17 @@ body {
-  //   width: 1200px;
-  // }  
+  @media screen and (min-width: 768px) {
+    width: 768px;
```
ширина тут не нужна, тем более фиксированная

MajaTrof replied:
ок

MajaTrof committed:
container

container

FrontKid commented on src/index.html:
```diff
@@ -1,18 +1,105 @@
-  <script type="module" src="./index.js"></script>
-</head>
+  <body>
+    <div class="wrapper"></div>
```
footer должен быть во wrapper

FrontKid commented on package.json:
```diff
@@ -18,7 +18,8 @@
   "dependencies": {
-    "modern-normalize": "^1.1.0"
+    "modern-normalize": "^1.1.0",
+    "update-browserslist-db": "^1.0.10"
```
Что это за пакет?

FrontKid commented on src/index.html:
```diff
@@ -1,18 +1,105 @@
+              >10340 Ridgeland Ave, Chicago Ridge, IL 60415, USA</a
+            >
+          </address>
+          <ul>
```
Не должно быть тегов без класса, классы по БЕМ методологии именуем

FrontKid commented on src/sass/2-components/_footer.scss:
```diff
@@ -0,0 +1,34 @@
+.footer {
+    width: 320px;
```
Не должно бытв фиксированой ширины, особенно в родителских блоках

FrontKid commented on src/sass/2-components/_footer.scss:
```diff
@@ -0,0 +1,34 @@
+
+    background-color: $dark-green;
+
+    @media screen and (min-width: $tablet-fixed) {
```
Медиа в самом низу файла должны быть

FrontKid commented on src/sass/2-components/_footer.scss:
```diff
@@ -0,0 +1,34 @@
+      }  
+    }
+
+// .footercard {
```
Комменты убираем все перед пушем, такого не должно быть в мейне

FrontKid commented on src/sass/2-components/_footer.scss:
```diff
@@ -0,0 +1,34 @@
+    
+// }
+
+.footercard__item {
```
Пробелов лищних не должно быть

FrontKid commented on src/sass/main.scss:
```diff
@@ -3,6 +3,7 @@
 /************************************************/
 @import "./1-helpers/variables";
 @import "./1-helpers/common";
+@import "./1-helpers/breakpoints";
```
Переменные должны быть в фалу _variables

FrontKid commented on src/sass/main.scss:
```diff
@@ -12,5 +13,6 @@
-/*============   COMPONENTS START   =============/
+/*============   COMPONENTS END   =============/
 /************************************************/
+@import "./2-components/footer";
```
Перемести на 14 строку, между СТАРТ И ЭНД

MajaTrof committed:
footer_mobile

З json забрала свої налаштування, в мене все працює без оновлення нпм
Є проблема в картках, вони довжиною 420, замість 412px. Я неможу найти де ці 8px зайві. Через це кнопки налазять на картки
Кнопки неготові, бо немає міксінів ще.

FrontKid commented on src/sass/1-helpers/_placeholders.scss:
```diff
@@ -0,0 +1,29 @@
+%text {
```
Общие стили для параграфа, ссылок и пунктов меню в фале _common

FrontKid commented on src/sass/main.scss:
```diff
@@ -3,14 +3,15 @@
 /************************************************/
 @import "./1-helpers/variables";
 @import "./1-helpers/common";
+@import "./1-helpers/placeholders";
```
Напиши стили для тегов

MajaTrof committed:
1

1

MajaTrof committed:
create footer.html

create footer.html

MajaTrof committed:
Merge branch 'main' into feature/footer

MajaTrof committed:
2

FrontKid commented on partials/footer.html:
```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
```
Это лишнаяя строка тут

MajaTrof replied:
у мене без цієї стрічки вибиває помилку, як це виправити?

FrontKid replied:
Какую ошибку

MajaTrof replied:
[{
	"message": "Doctype must be declared first.",
}]

MajaTrof committed:
footer

MajaTrof committed:
change media

MajaTrof committed:
1

MajaTrof committed:
.

FrontKid commented on partials/footer.html:
```diff
@@ -0,0 +1,187 @@
+        </button>
+      </li>
+    </ul>
+    <div class="footer-information">
```
У тебя несоответсвие блока с информацией на моб устройствах с макетом в тг скрин кинул

illia-dobro replied:
По завданню тут дві секції: contacts і footer

illia-dobro commented on partials/footer.html:
```diff
@@ -0,0 +1,187 @@
+          </li>
+        </ul>
+        <div class="footer-social__container">
+        <p class="footer-social__contact">
```
По макету зліва від номеру телефону

illia-dobro commented on partials/footer.html:
```diff
@@ -0,0 +1,187 @@
+        </ul>
+        <div class="footer-social__container">
+        <p class="footer-social__contact">
+          Feel free to contact </p>
```
По макету зліва від номеру телефону

MajaTrof committed:
Merge branch 'main' into feature/footer

MajaTrof committed:
buttons, changing the block of information

buttons, changing the block of information

FrontKid commented on partials/footer.html:
```diff
@@ -107,20 +102,27 @@ <h3 class="footercard__title">cafe</h3>
     <ul class="footer-button">
-      <li class="footer-button__item">
-        <button type="submit" class="footer-button__submit">
+        <button type="submit" class="footer-button__submit footer-button__submit--but1">
```
В  ul может быть только li элементы

MajaTrof committed:
ul-div
